python/python/glide/config.py

```diff
@@ -259,6 +258,11 @@ class ClusterClientConfiguration(BaseClientConfiguration):
             If the specified timeout is exceeded for a pending request, it will result in a timeout error. If not set, a default value will be used.
         client_name (Optional[str]): Client name to be used for the client. Will be used with CLIENT SETNAME command during connection establishment.
         protocol (ProtocolVersion): The version of the Redis RESP protocol to communicate with the server.
+        periodic_checks_interval (Optional[int]): Define the interval in seconds for periodic topology checks.
+            These checks evaluate changes in the cluster's topology, triggering a slot refresh when detected.
+            Periodic checks ensure a quick and efficient process by querying a limited number of nodes.
+            If no interval is set or the interval is 0, a default interval value will be used.
+            To disable periodic checks, pass -1 as the interval.
```

Comment:
this is confusing. let's use something clearer - instead of implicit negation values, let's use an object with two variants - either Manual period + value, or Disable periodic check.